### PR TITLE
Create home page

### LINKED ISSUE #7: Home Page
### Main Tasks:
- [x] Create header
- [x] Create 'search for a meal' section 
- [x] Create 'explore our kitchens' section
- [x] Link search icon with 'meal' page
- [x] Link every kitchen with 'country meals' page
- [x] Style the page
- [x] Make the page responsive 

**Expected time:** 5h

### Page Design:
02  Home

### PR DESCRIPTION
**Pull request by emanjaber**

### CODE REVIEW

**Review by mo7mad4 — DISMISSED** [review]
## Excellent

File: Html/home.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Cairo:wght@400;600;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/da6da1fff8.js" crossorigin="anonymous"></script>
+    <title>Home</title>
+</head>
+<body>
+    <img class="shape1" src="../assets/shape1.png">
+    <img class="shape2" src="../assets/shape2.png">
+    <img class="shape3" src="../assets/shape3.png">
+    <img class="shape4" src="../assets/shape4.png">
+    <img class="shape5" src="../assets/shape5.png">
```
Comment (sara219) [inline Html/home.html]: the folder called `Assets` not `assets`

Screenshot from 2021-04-14 00-12-26


Screenshot from 2021-04-14 00-17-25

**Review by sara219 — CHANGES_REQUESTED** [review]

File: Html/home.html (hunk 1 above)
Comment (emanjaber) [reply Html/home.html]: I modified it

**Review by sara219 — APPROVED** [review]

**Review by mo7mad4 — APPROVED** [review]
## Very Good Eman

**Review by RihamHasan — APPROVED** [review]

**Review by Ahmed-Sameer97 — APPROVED** [review]

**Pull request merged**